libmultilabel/utils.py

```diff
@@ -91,7 +91,7 @@ def set_seed(seed):
     if seed is not None:
         if seed >= 0:
             seed_everything(seed=seed)
-            torch.set_deterministic(True)
+            torch.use_deterministic_algorithms(True)
```

Comment:
So this is the new way to use deterministic settings?? I recall they issue a warning message?

Reply:
Yes. They rename `set_deterministic` to `use_deterministic_algorithms ` for more clarity. 
https://github.com/pytorch/pytorch/issues/49100

Reply:
Note: `use_deterministic_algorithms` get error on `1.7.1+cu101`.